runway/commands/modules_command.py

```diff
@@ -421,13 +424,16 @@ def run(self, deployments=None, command='plan'):  # noqa pylint: disable=too-man
                                     region)
                         LOGGER.info("Module options: %s", module_opts)
                         with change_dir(module_root):
-                            getattr(
-                                determine_module_class(module_root, module_opts)(  # noqa
-                                    context=context,
-                                    path=module_root,
-                                    options=module_opts
-                                ),
-                                command)()
+                            module_class = determine_module_class(module_root,
+                                                                  module_opts['class_path'])
+                            LOGGER.info("creating instance of '%s'", module_class)
```

Comment:
Let's make this LOGGER.debug, to keep the dotted class paths out of the normal logs

Reply:
Makes sense.   Done.